pycsob/utils.py

```diff
@@ -121,3 +121,31 @@ def get_card_provider(long_masked_number):
         if rx.match(long_masked_number[:6]):
             return provider_id, conf.CARD_PROVIDERS[provider_id]
     return None, None
+
+
+def to_camel_case(value):
+    """
+    If the value is not None, convert the value from snake_case to camelCase format. If the value is not in snake_case
+    format, return the original value.
+    """
+
+    if value is None:
+        return value
+
+    first_word, *other_words = value.split('_')
+    return "".join([first_word.lower(), *map(str.title, other_words)]) if other_words else first_word
+
+
+def convert_keys_to_camel_case(data):
```

Comment:
pokud se jedná o list, tak ten klíče nemá, asi bych to změnil na něco jako `convert_iterable_to_camel_case`